### ISSUE TITLE
Aria-hidden getting set and not unset

### ISSUE THREAD

**Issue by BoldBigflank** [sweetalert2/sweetalert2#2303]
## Current behavior

When Swal.fire happens twice, body children divs are given the aria-hidden label as well as the data-previous-aria-hidden is set to true, so after closing aria-hidden is set to true when originally it had no attribute.

## Expected behavior

When sweetalert sets the aria-hidden attribute, it should always return it to the original state when all are closed.

## Live demo 

https://codepen.io/boldbigflank/pen/YzVjWdG

---

Has SweetAlert2 helped you create an amazing application? You can show your support via GitHub Sponsors: https://github.com/sponsors/limonte

Alternative ways for donations (PayPal, cryptocurrencies, etc.) are listed here: https://sweetalert2.github.io/#donations

**Comment by limonte** [comment]
Thank you @BoldBigflank!

**Comment by leftlion** [comment]
Would it be possible to reopen this issue as it doesn't appear to have been fixed. I've created a code pen below to demonstrate the problem. I have a div with with aria-hidden set to true, and when one Swal popup is shown the div retains the aria-hidden attribute. However, when 2 Swal popups are shown the aria-hidden attribute is removed, which causes a secret message to be displayed.

If I were to set a 1 second timeout between the Swal popups the code would work ok.

Here is the code pen: https://codepen.io/leftlion/pen/JjOyWYM

Let me know if you'd prefer a new issue to be created instead. Thanks!
